music.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'item'
+require 'date'
+
+class MusicAlbum < Item
+  attr_accessor :name, :genres, :publish_date, :on_spotify
+  attr_writer :genre
+
+  def initialize(name, publish_date, on_spotify: false, genres: [])
+    super(publish_date)
+    @name = name
+    @publish_date = Date.parse(publish_date)
+    @on_spotify = on_spotify
+    @genres = genres
+  end
```

Comment:
- [ ] Remember that the relationship between `Item` and `Genre` is one-to-many, for instance, this `genres` array makes no sense here, as a Music Album belongs to only one Genre. Please consider fixing this! 🤓 🔧